vscode/src/workspace.ts

```diff
@@ -335,35 +339,32 @@ export class Workspace implements WorkspaceInterface {
     );
   }
 
-  private registerRebaseWatcher(context: vscode.ExtensionContext) {
+  private registerCreateDeleteWatcher(
+    context: vscode.ExtensionContext,
+    glob: string,
+  ) {
     const parentWatcher = vscode.workspace.createFileSystemWatcher(
-      new vscode.RelativePattern(
-        this.workspaceFolder,
-        "../.git/{rebase-merge,rebase-apply}",
-      ),
+      new vscode.RelativePattern(this.workspaceFolder, `../${glob}`),
     );
     const workspaceWatcher = vscode.workspace.createFileSystemWatcher(
-      new vscode.RelativePattern(
-        this.workspaceFolder,
-        ".git/{rebase-merge,rebase-apply}",
-      ),
+      new vscode.RelativePattern(this.workspaceFolder, glob),
     );
 
-    const startRebase = () => {
-      this.#rebaseInProgress = true;
+    const start = () => {
+      this.#inhibitRestart = true;
     };
-    const stopRebase = async () => {
-      this.#rebaseInProgress = false;
+    const stop = async () => {
+      this.#inhibitRestart = false;
       await this.restart();
     };
 
     context.subscriptions.push(
       workspaceWatcher,
       parentWatcher,
-      // When one of the rebase files are created, we set this flag to prevent restarting during the rebase
-      workspaceWatcher.onDidCreate(startRebase),
-      // Once they are deleted and the rebase is complete, then we restart
-      workspaceWatcher.onDidDelete(stopRebase),
+      // When one of the rebase/bisect files are created, we set this flag to prevent restarting during that action
```

Comment:
```suggestion
      // When one of the 'inhibit restart' files are created, we set this flag to prevent restarting during that action
```
(just so we're not maintaining the list in two places)